apm-lambda-extension/extension/process_env.go

```diff
@@ -23,6 +23,10 @@ import (
 	"os"
 	"strconv"
 	"strings"
+	"github.com/aws/aws-sdk-go/service/secretsmanager"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"encoding/base64"
```

Comment:
is there any sort of linting for import order? other projects separate out stdlib / elastic / 3rd party